conf/test.config

```diff
@@ -25,15 +25,21 @@ params {
     input  = "${projectDir}/assets/samplesheet.csv"
 
     // Fasta references
-    genomes_base = "s3://reference-data/genomes"
+    fasta                   = "https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-test-datasets/raw/main/data/genomics/homo_sapiens/genome/seq/SVcontrol/reference.fasta"
+    fai                     = "https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-test-datasets/raw/main/data/genomics/homo_sapiens/genome/seq/SVcontrol/reference.fasta.fai"
+    bwa                     = "https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-test-datasets/raw/main/data/genomics/homo_sapiens/genome/seq/SVcontrol/bwa.tar.gz"
+    expansionhunter_catalog = params.test_data["homo_sapiens"]["genome"]["expansionhunter"]
+    qdnaseq_male           = params.test_data["homo_sapiens"]["genome"]["genome_qdnaseq"]
+    qdnaseq_female         = params.test_data["homo_sapiens"]["genome"]["genome_qdnaseq"]
     igenomes_ignore = true
     genomes_ignore = false
-
-    vep_cache = null
+    genome = 'GRCh38'
+    genomes_base = "s3://reference-data/genomes"
 
     annotate = true
+    concat_output = true
 
     // Pipeline parameters
-    callers = "delly,manta,smoove" //manta,delly,smoove
+    callers = "delly,manta,smoove,gridss,expansionhunter,qdnaseq" //manta,delly,smoove,gridss,expansionhunter
```

Comment:
`gridss` niet meer default uitzetten?

Reply:
that's the test config. it runs well in the small tests, so I'd like to keep running it there. It's been removed from the actual defaults in `./nextflow.config`

Reply:
```
    callers = "manta,smoove,delly,expansionhunter,wisecondorx"

```